### PR TITLE
Bug/crypto defi vaults command broken #1822

### PR DESCRIPTION
**Pull request by JakubPluta**
# Description

- [ ] Summary of the change / bug fix.
Coindix changed their API spec. Now endpoint `https://apiv2.coindix.com/search` doesn't take as query parameter `risk` keyword, and also it's not returned in response. Also as get request to mentioned endpoint has flag:
`response = requests.get(..., verify=False)` it screams with urllib3 warnings. That`s why I also disabled this warning in stdout. For now it's in 
- [ ] Link # issue, if applicable.
https://github.com/OpenBB-finance/OpenBBTerminal/issues/1822
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
Before fix:
image

After fix.
image

# How has this been tested?

* I updated test cassetes, txt's, and csv's + i run manually tests.

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/defi/coindix_model.py [hunk 1]
```diff
@@ -4,15 +4,19 @@
...
 
+import urllib3
 import pandas as pd
 import requests
 
 from openbb_terminal.decorators import log_start_end
 from openbb_terminal.helper_funcs import get_user_agent
 
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Comment (JakubPluta) [inline openbb_terminal/cryptocurrency/defi/coindix_model.py:15]: To disable Insecure Request Warning I disabled it on coindix_model level, but maybe it should be moved to some more flags/config related place ?

**Comment by DidierRLopes** [comment]
Welcome back @JakubPluta 🚀 

For the tests you can do
```
pytest tests/openbb_terminal/cryptocurrency/test_cryptocurrency_helpers.py --record-mode=rewrite
```

File: openbb_terminal/cryptocurrency/defi/coindix_model.py (hunk 1 above)
Comment (DidierRLopes) [reply openbb_terminal/cryptocurrency/defi/coindix_model.py:15]: For now I think this looks good yea!

**Comment by JakubPluta** [comment]
> Welcome back @JakubPluta 🚀
> 
> For the tests you can do
> 
> ```
> pytest tests/openbb_terminal/cryptocurrency/test_cryptocurrency_helpers.py --record-mode=rewrite
> ```

Thanks Didier, I have to get used to the current structure of the project, as it changed a "little" from my last PR :)

**Comment by piiq** [comment]
the tests should pass once this https://github.com/OpenBB-finance/OpenBBTerminal/pull/1885 is merged

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**